ci/tekton/collections-build-task.yaml

```diff
@@ -41,6 +41,10 @@ spec:
           # install git
           yum install -y git which findutils
 
+          curl -L -s -o "appsody-0.5.4-1.x86_64.rpm" "https://github.com/groeges/appsody/releases/download/0.5.4/appsody-0.5.4-1.x86_64.rpm"
```

Comment:
Downloading the appsody CLI from this fork is only a temporary solution until the support for `buildah` is included in a proper release of `appsody`. Once this is available this will be updated to use that version.